api/openapi.yaml

```diff
@@ -4942,7 +4978,7 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/AvailabilityZoneEditData'
+              $ref: '#/components/schemas/AvailabilityZone'
```

Comment:
This is a breaking change introduced by https://phorge.dev.yugabyte.com/D26078, that should be reverted.